src/main/java/com/ibs/kpitrackerservice/service/UserService.java

```diff
@@ -1,7 +1,109 @@
 package com.ibs.kpitrackerservice.service;
 
+import com.ibs.kpitrackerservice.model.User;
+import com.ibs.kpitrackerservice.respository.TeamRepository;
+import com.ibs.kpitrackerservice.respository.UserRepository;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
 
+import java.util.List;
+import java.util.Optional;
+
 @Service
 public class UserService {
+;
+    private static final Logger LOGGER = LoggerFactory.getLogger(UserService.class);
+
+    @Autowired
+    private UserRepository userRepository;
+
+    @Autowired
+    private TeamRepository teamRepository;
+
+    public List<User> getAllUsers() {
+        return userRepository.findAll();
+    }
+
+    public User getUserByName(String name) {
+        try {
+            Optional<User> user = userRepository.findByEmployeeName(name);
+            if (!user.isEmpty() && user.isPresent()) {
+                LOGGER.info(Events.OPERATION_SUCCESSFUL.toString());
+                return user.get();
+            }
+            else {
+                LOGGER.error(Events.OPERATION_FAILED.toString()  + "; " + Events.NOT_FOUND.toString());
+            }
+        }
+        catch (Exception ex) {
```

Comment:
Are we really throwing any checked exception in this case? If this is just for the runtime exceptions, then we shouldn't handle it here.